helpers/interactive.js

```diff
@@ -0,0 +1,26 @@
+export const sharedInteractiveElems = {
```

Comment:
Nit: I'm guessing these are prefixed with "shared" to avoid naming conflict where they are imported eh? It seems implied already - given the choice I'd probably export these without the "shared" prefix, and have the consumer alias the import or name their modified version differently. 🤷‍♂️  Nevertheless, sharing them is great.